<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>1141</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="up_layout">
     <item>
      <widget class="QLabel" name="label_logo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>sprites/otros/logo_negro.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="stats_layout">
       <item>
        <widget class="QLabel" name="label_reputacion">
         <property name="text">
          <string>   Reputacion: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_dinero">
         <property name="text">
          <string>   Dinero: </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="round_layout">
         <item>
          <widget class="QLabel" name="label_ronda">
           <property name="text">
            <string>   Ronda: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boton_comenzar_ronda">
           <property name="text">
            <string>Comenzar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="clientes_layout">
       <item>
        <widget class="QLabel" name="label_clientes_atendidos">
         <property name="text">
          <string>   Atendidos: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_clientes_perdidos">
         <property name="text">
          <string>   Perdidos: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_clientes_proximos">
         <property name="text">
          <string>   Proximos: </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="botones_layout">
       <item>
        <widget class="QPushButton" name="boton_pausa">
         <property name="text">
          <string>Pausa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boton_salir">
         <property name="text">
          <string>Salir</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_MiuEnzo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>800</y>
      <width>20</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/mesero/up_01.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_mapa">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>891</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/mapa/mapa_con_borde_2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>959</x>
      <y>169</y>
      <width>221</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="tienda_layout">
     <item>
      <widget class="QLabel" name="label_tienda">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Tienda</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_comenzar_ronda</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>comenzar_ronda()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>824</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_pausa</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pausa()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1033</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>824</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_salir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>salir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1033</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>824</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>comenzar_ronda()</slot>
  <slot>pausa()</slot>
  <slot>salir()</slot>
 </slots>
</ui>
